### DISCUSSION TITLE
[Docs] 如何追溯历史版本的文档

### DISCUSSION THREAD

**Discussion by Otto-J** [element-plus/element-plus#10902] [Q&A] [answered]
Feature type: **`Docs`**

## Details

- Enhanced component name: `N/A`

## Description

目前的官方文档默认是最新版本，请问如何找到历史版本的文档，比如 1.0 1.1 系列的版本文档，可以通过有规律的网址参数访问吗？比如 vercel 地址携带参数。
进一步地，是否可以考虑同步补充类似版本切换的功能？

### Additional comments

_(empty)_

**Comment by chenxch** [comment, marked as answer]
目前因为资源有限，线上只有最新的文档，如果你不介意查看英文文档，可以clone仓库，切到版本tag分支运行docs。如果实在需要查看中文稳定，需要查看gh-pages的历史记录，自己找到版本发布日期的分支查看。

**Reply by Otto-J** [reply]
clone 仓库，自行部署对应版本的文档

这个可以解决问题，感谢
